src/types/insights.ts

```diff
@@ -46,7 +46,8 @@ export type InsightsClient = InsightsSendEvent &
   InsightsOnUserTokenChange &
   InsightsGetUserToken &
   InsightsInit &
-  InsightsSetUserToken & {
+  InsightsSetUserToken &
+  InsightsGetUserToken & {
```

Comment:
Isn't this a duplicate with line 47?

Reply:
Good catch https://github.com/algolia/instantsearch.js/pull/4724/commits/ad13acb2b833d045ba3dbc724f01c2637061511b